database/src/main/resources/db/migration/V1_82__gtfs.sql

```diff
@@ -0,0 +1,40 @@
+ALTER TABLE "gtfs-stop"
+  ADD COLUMN "wheelchair-boarding" INTEGER CHECK ("wheelchair-boarding" BETWEEN 0 AND 2);
+
+DROP TABLE "gtfs-trip";
+
+CREATE TYPE "gtfs-stop-time-info" AS (
+  "trip-id"           TEXT,
+  "arrival-time"      INTERVAL,
+  "departure-time"    INTERVAL,
+  "stop-sequence"     INTEGER,
+  "stop-headsign"     TEXT,
+  "pickup-type"       INTEGER,
+  "drop-off-type"     INTEGER,
+  "shape-dist-traveled" NUMERIC
+);
+
+CREATE TYPE "gtfs-trip-info" AS (
+  "trip-id"               TEXT,
```

Comment:
This links gtfs_trip to this type? gtfs_trip.id is int type. This should be too.

Reply:
trip_id can contain characters.